csp_billing_adapter/adapter.py

```diff
@@ -141,60 +143,63 @@ def event_loop_handler(
 ) -> datetime.datetime:
     """Perform the event loop processing actions."""
     log.info('Starting event loop processing')
+    now = get_now()
```

Comment:
I had meant to ask previously if there is any special meaning associated with this "now" value, other than the time the event loop runs at?